android/src/main/java/com/foursquare/movement/react/Utils.kt

```diff
@@ -49,7 +49,7 @@ internal object Utils {
     }
 
     private fun geofenceEventJson(geofenceEvent: GeofenceEvent) = Arguments.createMap().apply {
-        putString("id", geofenceEvent.id)
+        putString("geofenceId", geofenceEvent.id)
```

Comment:
updated for this https://github.com/foursquare/movement-sdk-react-native/pull/3/files#diff-619cc2239a7eb4ef05041767e6ab0e4fb069e98ee9d2c83c2d8e1865c475e141R10